bin/compile

```diff
@@ -35,24 +35,20 @@ ENV_DIR=$3
 # Export Path variables, for use in sub-scripts.
 export BUILD_DIR CACHE_DIR ENV_DIR
 
-# Set the Buildpack's internet target for downloading Python distributions.
-# The user can provide BUILDPACK_VENDOR_URL to specify a custom target.
-# Note: this is designed for non-Scalingo use, as it does not use the user-provided
+# Set the base URL for downloading buildpack assets like Python runtimes.
+# The user can provide BUILDPACK_S3_BASE_URL to specify a custom target.
+# Note: this is designed for non-Heroku use, as it does not use the user-provided
 # environment variable mechanism (the ENV_DIR).
-# VENDOR_URL="https://lang-python.s3.amazonaws.com/$STACK"
-VENDOR_URL="https://python-binaries.scalingo.com/$STACK"
-if [[ -n ${BUILDPACK_VENDOR_URL:-} ]]; then
-    VENDOR_URL="$BUILDPACK_VENDOR_URL"
-elif [[ -n ${USE_STAGING_BINARIES} ]]; then
-    VENDOR_URL="$USE_STAGING_BINARIES/$STACK"
-fi
-export VENDOR_URL
+# S3_BASE_URL="${BUILDPACK_S3_BASE_URL:-"https://heroku-buildpack-python.s3.amazonaws.com"}"
+S3_BASE_URL="${BUILDPACK_S3_BASE_URL:-"https://python-binaries.scalingo.com"}"
```

Comment:
We don't have a Notion page about how to upload our own binaries here. Can you create one?